### PR TITLE
Implemented security manager check fixed other checks

### PR DESCRIPTION
**Pull request by kharyam**
Implemented security manager check, fixed the transfer log check and vendor supported check.

### CODE REVIEW

**Comment by shawndwells** [comment]
@openscap-ci add to whitelist

**Comment by shawndwells** [comment]
@openscap-ci ok to test

File: JBoss/EAP/6/templates/static/oval/jboss_eap_vendor_supported.xml [hunk 1]
```diff
@@ -3,8 +3,12 @@
     <metadata>
       <title>JBoss Enterprise Application Platform Supported Version</title>
       <description>Installed version of JBoss is a vendor supported version.</description>
+      <affected family="undefined">
+        <platform>JBoss Enterprise Application Platform 6</platform>
+      </affected>
+      <reference source="CPE" ref_id="cpe:/a:redhat:jboss_enterprise_application_platform:6.0.0" />
```
Comment (redhatrises) [inline JBoss/EAP/6/templates/static/oval/jboss_eap_vendor_supported.xml]: @kharyam please remove the CPE reference. This should be handled by the `<extended_definition />`.

Comment (kharyam) [reply JBoss/EAP/6/templates/static/oval/jboss_eap_vendor_supported.xml]: @redhatrises removed, thanks

**Comment by redhatrises** [comment]
Thanks ack.

**Pull request merged**